vnext/Desktop/ABI/NativeTracing.idl

```diff
@@ -1,4 +1,4 @@
-namespace facebook.react {
+namespace Microsoft.React {
```

Comment:
>Microsoft.React [](start = 10, length = 15)

If we change the namespace, then it would be better to use the Microsoft.ReactNative to keep it in sync with the Microsoft.ReactNative.dll

Reply:
Microsoft.ReactNative.Desktop ?

Reply:
I believe there was a potential conflict issue if the same namespace was used. Is that not accurate?

@asklar We already have several types under namespace `Microsoft::React` before `Microsoft::ReactNative` was introduced.
Adding a third namespace would make things even more confusing.

I'd like to follow up offline for guidance regarding the definitive namespaces to use.
For now, it's very clear that using `facebook::react` for non-Facebook sources is wrong.

Reply:
We are working towards common "core dll" to be shared between UWP and Win32. It would be better if we use the same namespace to help us there. We use the Microsoft.ReactNative namespace today. It may be changed in future, but for now it would be nice to be consistent.

---
In reply to: [430837209](https://github.com/microsoft/react-native-windows/pull/5012#discussion_r430837209) [](ancestors = 430837209)

Reply:
As discussed offline, I'll scope down the change and remove any namespace renamings.